podcasts/Onboarding/Plus/PlusButtonStyles.swift

```diff
@@ -41,9 +47,23 @@ struct PlusGradientFilledButtonStyle: ButtonStyle {
 struct PlusGradientStrokeButton: ButtonStyle {
     let isSelectable: Bool
     let isSelected: Bool
+    let plan: Constants.Plan
+
+    private var foregroundColor: Color {
+        plan == .plus ? Color.plusGradientColor1 : Color(hex: "AFA2FA")
```

Comment:
Can we add this to Color extension?

Reply:
Done in 0ae9c2bca75d70f141e89dfd0f14a8ac039451c3